remix.init/index.js

```diff
@@ -173,6 +173,7 @@ const main = async ({ isTypeScript, packageManager, rootDirectory }) => {
     stdio: "inherit",
   });
 
+  // Create a starting .env file?
```

Comment:
one should just be in the repo.

Reply:
But, you've excluded them! Should we generate a file that won't be picked up in git? 😕